history.rst

```diff
@@ -5,6 +5,26 @@
 OMERO version history
 =====================
 
+5.4.10 (November 2018)
+----------------------
+
+This is a bug-fix release.
+
+Import improvements include:
+
+- insight: improve saving of ROIs and Measurements when used as a ImageJ plugin
```

Comment:
I find this confusing. Insight-ij plugin is shipped as a separate entity from insight. It serves a different purpose. The knowledge that "this is just insight" is an insider-knowledge. You cannot "use insight as ImageJ plugin" as formulated here (and as very often happens when people attempt to install the plugin).

Suggest have a separate ImageJ/Fiji plugin section